src/pages/modules/TimeSeriesPage.tsx

```diff
@@ -1,3 +1,5 @@
+import TimeSeriesComponent from '@/components/TimeSeries/TimeSeriesComponent';
```

Comment:
I personally don't like adding `Component` to the end of the component's name. Everything that you render are components and I don't think we need to add `Component` suffix to their names.